apps/journeys-admin/src/components/JourneyVisitorsList/FilterDrawer/GoogleSheetsSyncDialog/GoogleSheetsSyncDialog.tsx

```
@@ -0,0 +1,1393 @@
+  function handleRequestDeleteSync(syncId: string): void {
+    setSyncIdPendingDelete(syncId)
+  }
+
+  const isGoogleActionDisabled = integrationsData == null
+
+  const validationSchema = object().shape({
+    integrationId: string().required(t('Integration account is required')),
+    sheetName: string().required(t('Sheet tab name is required')),
+    spreadsheetTitle: string().when('googleMode', {
+      is: 'create',
+      then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)